person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age
+    @age > 18
+  end
+
+  private :of_age
+
+  def can_use_services?
+    return true if of_age || @parent_permission
```

Comment:
- [ ] **[OPTIONAL]** I think it would make more sense to rename `of_age` to `of_age?` based on convention since it returns `true` or `false`

- [ ] **[OPTIONAL]** We can save some code here on line 20:

  ```
  of_age? || @parent_permission
  ```